cmd/importer/db.go

```diff
@@ -46,6 +46,9 @@ func intRangeValue(column *column, min int64, max int64) (int64, int64) {
 }
 
 func randStringValue(column *column, n int) string {
+	if column.hist != nil {
+		return column.hist.randString(n)
```

Comment:
Here _n_ is the maximum length of the rand string, but `column.hist.randString(n)` seems to return the rand string with length _n_ exactly.